msg/odlc_shape.msg

```diff
@@ -0,0 +1 @@
+int8 shape
```

Comment:
This is one where I am ok having a single type with constants in it to signify each shape